src/main/java/org/openjdk/jextract/impl/Logger.java

```diff
@@ -50,27 +51,58 @@ public Logger(PrintWriter outWriter, PrintWriter errStream) {
         this.errWriter = errStream;
     }
 
+    public void print(ClangException exp) {
+        errWriter.println(String.format("%1$s: %2$s: %3$s",
```

Comment:
In all these cases the "numbered" format argument is probably not necessary, as one can easily see which is which.

Reply:
You're right. It was number earlier even when it had single arg in the same file. Followed it for consistency.